### PR TITLE
The consul-template system service is run as the root user

### LINKED ISSUE #55: The consul-template system service is run as the root user
It would be more secure to allow the option of creating a specific "consul-template" user and group and use these for file permissions and for running the system service.

### PR DESCRIPTION
**Pull request by awarner-greshamtech**
Hello, this is a PR which fixes #55 to allow the user and group that is used to run the service to be optionally specified, and optionally created by this ansible role. Where the user and group are created by this role, it is a non-root user.

### CODE REVIEW

File: molecule/_shared/converge.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+- name: Converge
+  hosts: all
+  become: true
+  become_user: root
+  tasks:
+    - name: "Include configure_nomad_server"
```
Comment (griggheo) [inline molecule/_shared/converge.yml:7]: Not sure why the reference to nomad is used here. Can you please clarify?

**Review by griggheo — CHANGES_REQUESTED** [review]

File: molecule/_shared/converge.yml (hunk 1 above)
Comment (awarner-greshamtech) [reply molecule/_shared/converge.yml]: A type, I have fixed

**Comment by awarner-greshamtech** [comment]
I have the molecule tests running on github actions ( https://github.com/awarner-greshamtech/ansible-consul-template/actions/workflows/main.yml ) would you be interesting in a PR for that workflow?

**Review by griggheo — APPROVED** [review]

**Comment by griggheo** [comment]
@awarner-greshamtech yes please create a PR for the molecule tests in GH Actions.

**Pull request merged**